microsoft/testsuites/dpdk/dpdkvpp.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import time
+from typing import List, Type
+
+from assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian, Fedora
+from lisa.tools import Gcc, Git, Make, Modprobe
+from lisa.util import UnsupportedDistroException
+
+
+class DpdkVpp(Tool):
+
+    VPP_SRC_LINK = "https://github.com/FDio/vpp.git"
+    REPO_DIR = "vpp"
+
+    @property
+    def command(self) -> str:
+        return "vpp"
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Gcc, Make, Git]
```

Comment:
It looks they are not used in this tool, so it's not necessary. The dependencies will be installed, if they are added here. It's unnecessary and slow down the test case.

Reply:
They are used but are used by the install script VPP provider that we end up using. It looks weird but they are needed to run the script.

Reply:
I see, please comment above too. It prevents others remove it suddenly in future.